ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -366,6 +366,18 @@
   public static final String KSQL_VARIABLE_SUBSTITUTION_ENABLE_DOC
       = "Enable variable substitution on SQL statements.";
 
+  public static final String KSQL_RETRIABLE_REQUESTS_MAX_RETRIES
```

Comment:
I haven't looked at the code yet, but I wanted to comment that I don't think we should set the default behavior to retry or sleep. For pull queries, I suspect a better default behavior is to fail fast than to hang - so that the user can decide whether they want to retry or not. What's important is to make sure that we have a good error message returned to the user.